src/docs/truffle/reference/configuration.md

```diff
@@ -92,8 +92,10 @@ networks: {
     port: 80,
     network_id: 1,        // Ethereum public network
     // optional config values:
-    // gas
-    // gasPrice
+    // gas                  -
+    // gasPrice             - use gas and gasPrice if creating type 0 transactions
+    // maxFeePerGas         -
+    // maxPriorityFeePerGas - use maxFeePerGas and maxPriorityFeePerGas if creating type 2 transactions
```

Comment:
It only explains the defaults for these values (it describes the params that have defaults). As far as I know there are no defaults for these values. Maybe it could be useful to link the eip in the comments